set.mm

```diff
@@ -472151,6 +472186,240 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Sumset / Minkowski sum
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  The sumset (also called the Minkowski sum) of two subsets ` A ` and ` B ` ,
+  is defined to be the set of all sums of an element from ` A ` with an element
+  from ` B ` .
+
+  For an operation ` .+ ` This can be expressed as ` ( .+ " ( A X. B ) ) ` .
```

Comment:
Can you comment on this new notation for a sumset vs the already existing `LSSum`? The latter works for any group, and can be used with a more readable (in my opinion) notation
```
.(+) = ( LSSum ` G )
...
( A .(+) B )
```
rather than ` ( .+ " ( A X. B ) ) `.

Reply:
Ok, this is finally resolved with the last commit!